gcn/notices/core/Attitudes.schema.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://gcn.nasa.gov/schema/gcn/notices/glowbug/Attitudes.schema.json",
+  "title": "Instrument and Spacecraft Attitudes",
+  "description": "Enables GCN producers to submit instrument and spacecraft attitude data.",
+  "type": "object",
+  "properties": {
+    "instrument_attitude": {
+      "$ref": "/schema/gcn/notices/core/AttitudeBase.schema.json",
+      "description": "Instrument attitude and rotaional rates."
+    },
+    "spacecraft_attitude": {
+      "$ref": "/schema/gcn/notices/core/AttitudeBase.schema.json",
+      "description": "Spacecraft attitude and rotaional rates."
```

Comment:
We've realized that we cannot nest the same schema multiple times, as it doesn't work in the flat data file, because the two sets of records would look identical.

Reply:
I don't think that's right. You can have multiple subrecords of the same type. (On the other hand, it would be meaningless to create a _union_ of multiple copies of the same type.)

I think that this is another case of confusion between inheritance ("is-a") versus membership ("has-a") relationships.